js/plugins/chroma/src/index.ts

```diff
@@ -151,7 +152,7 @@ export function chromaRetriever<EmbedderCustomOptions extends z.ZodTypeAny>(
         include: getIncludes(options?.include),
         where: options?.where,
         whereDocument: options?.whereDocument,
-        queryEmbeddings: embedding,
+        queryEmbeddings: embedding[0].embedding,
```

Comment:
I'm thinking we should throw if `embedding.length !== 1`

Reply:
(Follow up PR)